src/routes/jira/server/jira-ghe-server-apps.ts

```diff
@@ -0,0 +1,35 @@
+import { NextFunction, Request, Response } from "express";
+import { GitHubServerApp } from "models/github-server-app";
+
+export const JiraGheServerApps = async (
+	req: Request,
+	res: Response,
+	next: NextFunction
+): Promise<void> => {
+	try {
+		req.log.debug("Received Jira GHE server apps page request");
+
+		const { id: installationId } = res.locals.installation;
+		const githubServerBaseUrl = req.query.serverUrl as string;
+
+		if (!githubServerBaseUrl) {
+			throw new Error("No server URL passed!");
+		}
+
+		const servers = await GitHubServerApp.getAllForGitHubBaseUrl(githubServerBaseUrl, installationId) || [];
+		// `identifier` is the githubAppName for the GH server app
+		const serverApps = servers.map(server => ({ identifier: server.gitHubAppName }));
```

Comment:
Wouldn't we just call it `gitHubAppName`? Imo, if you have to explain why you gave a variable/func a name, then you need to rename it. The name itself should provide the explanation.

Reply:
This data is to be passed to the `select-table` component. And this component displays the list provided to it, which could be GHE servers or server apps.
For GHE server, we are displaying the _baseUrl_ as the identifier, whereas for GH server apps, we are displaying the _gitHubAppName_ as the identifier.
This property had to be named something generic, so named it `identifier`. Confirmed with @gxueatlassian.